modules/periphery/metadata.json

```diff
@@ -11,6 +11,7 @@
         "github:peripheryapp/periphery"
     ],
     "versions": [
+        "3.0.0",
```

Comment:
Bazel will likely sort this below the beta versions (which should have used `-beta` instead of `.beta`). 3.0.0.1 or 3.0.0.bcr.1 would work better.

Reply:
The opposite was true actually, I moved it to the top because one of the checks failed with it at the bottom.

Reply:
The sorting in this file is correct, but it's likely not what you want: If one Bazel module depends on 3.0.0.beta1 and another on 3.0.0, the former will be chosen as the highest version.

Reply:
Ah I see. Would it be acceptable to yank the betas instead?

Reply:
Yeah, that's a good approach!